chrome/content/zotero/xpcom/quickCopy.js

```diff
@@ -321,32 +331,35 @@ Zotero.QuickCopy = new function() {
 				}
 				
 				var content = {
-					text: contentType == "html" ? html : text,
+					text: format.contentType == "html" ? html : text,
 					html: copyHTML
 				};
 				
 				return content;
 			}
 			
+			// determine locale preference
+			var locale = format.locale ? format.locale : Zotero.Prefs.get('export.quickCopy.locale');
+			
```

Comment:
> A mismatch might not actually cause any problems though, so maybe don't worry about it.

It shouldn't. (and these mismatches are very common with the current `extensions.zotero.export.bibliographyLocale` pref and styles' default-locales, but the latter always take priority)